test/runtests.jl

```diff
@@ -331,7 +340,8 @@ end
 end
 
 # "quine" dataset discussed in Section 7.4 of "Modern Applied Statistics with S"
-quine = dataset("MASS", "quine")
+# quine = dataset("MASS", "quine")
```

Comment:
This should work once https://github.com/JuliaLang/METADATA.jl/pull/17366 has been merged and that will most likely happen later today so please uncomment this and delete the csv.

Reply:
Okay.  I have a commit with the updates locally.  I'll push once the metadata pr is merged so it'll retrigger the tests.

Reply:
Okay. The new version of RDatasets has been merged so this should work now.